### ISSUE TITLE
pcntl extension is missing/not enabled in image

### ISSUE THREAD

**Issue by akovalyov** [cloudflare/docker-phan#3]
Hi,

thank you for this cool tiny image.
While playing with it, I have discovered that it is impossible to use `-j`(aka `--processes`) option.
Staketrace should be self-explanatory

``` bash
$ docker run -v $PWD:/mnt/src --rm -u "$(id -u):$(id -g)" cloudflare/phan:latest -j=2
Error: Call to undefined function Phan\pcntl_fork() in /opt/phan/src/Phan/ForkPool.php:53
Stack trace:
#0 /opt/phan/src/Phan/Phan.php(160): Phan\ForkPool->__construct(2, Array, Object(Closure), Object(Closure), Object(Closure))
#1 /opt/phan/src/phan.php(42): Phan\Phan::analyzeFileList(Object(Phan\CodeBase), Array)
#2 /opt/phan/phan(2): require_once('/opt/phan/src/p...')
#3 {main}
```

**Comment by fruitl00p** [comment]
@akovalyov this should have been fixed by #2 ;) (and my testing confirms this?)

**Comment by akovalyov** [comment]
@fruitl00p thank you for feedback. I had an old version of image cached on my machine. docker `latest` tag was confusing. After I pulled latest `latest`, everything works. Thanks again!
